data/input_files/tests/thermal_conduction/dyn_exp_solve.lua

```diff
@@ -51,7 +51,7 @@ thermal_conduction = {
     order = 2,
 
     -- material parameters
-    kappa = 0.5,
+    materials = { { model = "LinearIsotropicConductor", kappa = 0.5, cp = 0.5, density = 0.5 }, },
```

Comment:
I didn't see any tests with the `LinearConductor` model (with matrix-valued `kappa`), so it might be good to add another test for those materials

Reply:
I agree. All I really did was modify the previous lua files to work with the new material schema. We also need examples for all solid mechanics materials besides Neo-hookean.

Reply:
The additional tests should probably go in the integration test follow on PR.